src/constants/routing.ts

```diff
@@ -133,9 +135,15 @@ export const COMMON_BASES: ChainCurrencyList = {
     ExtendedEther.onChain(SupportedChainId.POLYGON),
     WRAPPED_NATIVE_CURRENCY[SupportedChainId.POLYGON],
   ],
+  [SupportedChainId.POLYGON]: [
+    ExtendedEther.onChain(SupportedChainId.POLYGON),
+    WRAPPED_NATIVE_CURRENCY[SupportedChainId.POLYGON],
+    WETH_POLYGON,
```

Comment:
should we add usdc here as well?

Reply:
and dai too maybe

Reply:
yes please, all the usual:  ethereum logo
ETH (n/a on polygon)
DAI
USDC
USDT
WBTC
WETH